### DISCUSSION TITLE
Dutch translation

### DISCUSSION THREAD

**Discussion by rdwdm** [dolezsa/thermal_comfort#147] [General]
{
  "state": {
    "thermal_comfort__frost_risk": {
      "no_risk": "Geen",
      "unlikely": "Onwaarschijnlijk",
      "probable": "Waarschijnlijk",
      "high": "Hoogstwaarschijnlijk"
    },
    "thermal_comfort__thermal_perception": {
      "dry": "Droog",
      "very_comfortable": "Erg aangenaam",
      "comfortable": "Aangenaam",
      "ok_but_humid": "Goed, maar vochtig",
      "somewhat_uncomfortable": "Minder aangenaam",
      "quite_uncomfortable": "Erg vochtig, niet aangenaam",
      "extremely_uncomfortable": "Erg onaangenaam, benauwd",
      "severely_high": "Erg hoog, gevaarlijk voor astma gerelateerde ziekten"
    },
    "thermal_comfort__simmer_zone": {
      "cool": "Koel",
      "slightly_cool": "Beetje koel",
      "comfortable": "Aangenaam",
      "slightly_warm": "Beetje warm",
      "increasing_discomfort": "Toenemend onbehagelijk",
      "extremely_warm": "Extreem warm",
      "danger_of_heatstroke": "Gevaar voor zonnesteek",
      "extreme_danger_of_heatstroke": "Extreem gevaar voor een zonnesteek",
      "circulatory_collapse_imminent": "Gevaar voor de bloedsomloop"
    }
  }
}

**Comment by dolezsa** [comment]
thank you!
